datahub-web-react/src/app/entity/ownership/table/OwnershipTable.tsx

```diff
@@ -55,6 +55,10 @@ export const OwnershipTable = ({ ownershipTypes, setIsOpen, setOwnershipType, re
                 emptyText: <Empty description="No Ownership Types found!" image={Empty.PRESENTED_IMAGE_SIMPLE} />,
             }}
             pagination={false}
+            style={{
+                height: 'calc(100% - 30px)',
```

Comment:
Why are we computing the height of the ownership table here? Can we try to use display: flex and flex-direction:column on the parent components for settings to see if the settings content can continue on down the page?

Reply:
after setting  height: calc(100% - 11vh);  to MangeOwnerShip.tsx notneed to add height just overflow can add the scroll.